gomodfs.go

```diff
@@ -65,12 +65,21 @@ type FS struct {
 
 	Verbose bool
 
+	// FileCacheSize specifies the file cache size to use.
+	// If zero, a default size is used.
+	FileCacheSize int64
```

Comment:
So far unused, but for NFS-based gomodfss will be used to set the Cache.MaxSize?